.tx/config

```diff
@@ -0,0 +1,8 @@
+[main]
+host = https://www.transifex.com
+
+[edx-platform.your-resource-name-here]
```

Comment:
Are we supposed to replace the `your-resource-name-here` with something that points to `frontend-app-learner-record`? I was just looking [at the one in `frontend-app-discussions`](https://github.com/openedx/frontend-app-discussions/blob/97b92a176241e4ce67ac0dafe31b31b61050f98e/.tx/config#L4)

Reply:
👀 good eyes! I noticed this bad copy-pasta as you were looking at it too.

Reply:
The other MFEs have it formatted like this `o:open-edx:p:edx-platform:r:frontend-app-discussions`. Do we need that same prefix? I haven't been able to parse what it's doing

Reply:
I think we'll need to update that when we move this repo to the open-edx org, but it's still in our org, so I don't think we need to do that.

Reply:
Eh, I'll update it anywho. We can always fix it if needed.